source/Jobbr.Server/JobbrServer.cs

```diff
@@ -11,213 +11,211 @@
+
+            var assemblies = new List<Assembly> { GetType().Assembly };
+            assemblies.AddRange(_jobbrComponents.Select(component => component.GetType().Assembly));
+
+            assemblies.Distinct().ToList().ForEach(assembly => { sb.AppendLine($"{assembly.ManifestModule.Name} {assembly.GetVersion()}"); });
+
+            _logger.LogInformation("{assemblies}", sb.ToString().TrimEnd());
+        }
+
         private void StartInternalComponents()
         {
             try
             {
-                Logger.DebugFormat("Starting Scheduler ({0})...", this.scheduler.GetType().Name);
-                this.scheduler.Start();
+                _logger.LogDebug("Starting Scheduler ({0})...", _scheduler.GetType().Name);
+                _scheduler.Start();
 
-                Logger.DebugFormat("Starting Executor ({0})...", this.executor.GetType().Name);
-                this.executor.Start();
+                _logger.LogDebug("Starting Executor ({0})...", _executor.GetType().Name);
+                _executor.Start();
 
-                Logger.Info("All services (Scheduler, Executor) have been started successfully.");
+                _logger.LogInformation("All services (Scheduler, Executor) have been started successfully.");
             }
             catch (Exception e)
             {
-                Logger.FatalException("A least one service couldn't be started. Please see the exception for details.", e);
+                _logger.LogCritical("A least one service couldn't be started. Please see the exception for details. '{exception}'", e);
             }
         }
 
         private void StartOptionalComponents()
         {
-            if (this.components == null)
+            if (_jobbrComponents == null)
             {
                 return;
             }
 
             try
             {
-                foreach (var jobbrComponent in this.components)
+                foreach (var jobbrComponent in _jobbrComponents)
                 {
                     var type = jobbrComponent.GetType();
-                    Logger.DebugFormat($"Starting JobbrComponent '{type.FullName}' ...");
+                    _logger.LogDebug("Starting JobbrComponent '{fullName}' ...", type.FullName);
                     jobbrComponent.Start();
                 }
 
-                Logger.Info("All Optional Services have been started successfully.");
+                _logger.LogInformation("All Optional Services have been started successfully.");
 
-                this.isRunning = true;
+                _isRunning = true;
             }
             catch (Exception e)
             {
-                Logger.FatalException($"A least one service couldn't be started. Reason: {e}\n\n Please see the exception for details.", e.InnerException);
+                _logger.LogCritical("At least one of the services couldn't be started. Reason: {e}\n\n Please see the exception for details.", e.InnerException);
```

Comment:
Pass the whole exception to the log: `_logger.LogCritical(e, "message", values)`